### DISCUSSION TITLE
Getting error : "404- not found" on the localhost and "compiled not found in (258ms)"

### DISCUSSION THREAD

**Discussion by kkunal026** [vercel/next.js#63061] [Help]
### Summary

I am trying to make a NextJs project in which I am making a profile page. When I am putting the 'page.tsx' inside the root -> profile -> [id] -> page.tsx, its showing page not found on the localhost but if I do the same thing by putting the 'page.tsx' inside the root-> profile -> page.tsx its working fine. I can't seem to fix the error. Please help me with this.

![image](https://github.com/vercel/next.js/assets/115064201/207be27b-a94e-40dc-939d-8807f730d7c0)
![image](https://github.com/vercel/next.js/assets/115064201/925de9a3-7d56-4937-9889-73b98666fe92)

**Comment by DevDaniloFerrari** [comment]
Perhaps you are not including the ID in the URL; try accessing it this way.
![image](https://github.com/vercel/next.js/assets/40414119/2e956164-a0e1-450d-93d7-2abb0a15eeae)

**Reply by kkunal026** [reply]
The code has to check for the id itself...

![image](https://github.com/vercel/next.js/assets/115064201/71da7633-a472-4c59-b490-8ef378ed1bba)

**Comment by DevDaniloFerrari** [comment]
The 'next' already handles the URL, and if you pass 'profile/' to it, it will redirect to 'profile.' You don't need to implement this handling for 'id == null.
